package.json

```diff
@@ -13,7 +13,8 @@
     "publish-docs": "gh-pages -d docs/jsdocs",
     "start:test": "gulp dev:test",
     "build:test": "gulp build:test",
-    "test": "yarn audit && yarn install-test-deps && yarn test:unit && yarn lint",
+    "test:ci": "yarn install-test-deps && yarn test:unit && yarn lint && node scripts/audit.js",
+    "test": "yarn test:unit && yarn lint && yarn audit",
```

Comment:
did you mean to remove `yarn install-test-deps` from this command?

Reply:
@diracdeltas no it was my mistake, added back in and re-organized 👍